jwt/api.py

```diff
@@ -25,6 +25,25 @@ def __init__(self, algorithms=None):
             if key not in self._valid_algs:
                 del self._algorithms[key]
 
+        if not options:
+            options = {}
+
+        self.default_options = {
+            'verify_signature': True,
+            'verify_exp': True,
+            'verify_nbf': True,
+            'verify_iat': True,
+            'verify_aud': True,
```

Comment:
How many people will choose to solve, non-verification messages by figuring out they can set all of these to False rather than taking the time to understand what it all means?